fix-AppData-file.patch

```diff
@@ -1,13 +1,23 @@
 diff --git a/share/metainfo/torbrowser.appdata.xml b/share/metainfo/torbrowser.appdata.xml
-index 5577868..dfc51fe 100644
+index 5577868..f4b6d67 100644
 --- a/share/metainfo/torbrowser.appdata.xml
 +++ b/share/metainfo/torbrowser.appdata.xml
 @@ -1,7 +1,7 @@
  <?xml version="1.0" encoding="UTF-8"?>
  <!-- Copyright 2014 Micah Lee <micah@micahflee.com> -->
- <component>
+-<component>
 - <id>org.torproject.torbrowser.desktop</id>
++<component type="desktop">
 + <id>torbrowser.desktop</id>
```

Comment:
As you're already patching the appdata file, any reason for not setting the right id here?

Reply:
Upstream don't use the _com.github.micahflee.torbrowser-launcher_ id and I am sending all patches upstream + would like to drop them in next releases.